common/production_methods/07_government.txt

```diff
@@ -407,7 +407,128 @@ pm_religious_academia = {
+		level_scaled = {
+			building_employment_officers_add = 150
+            building_employment_academics_add = 50
+            building_employment_soldiers_add = 250
+		}
+	}
+	building_modifiers = {
+		workforce_scaled = {
+			building_input_paper_add = 0.5
+			building_input_small_arms_add = 1
+		}	
+	}
+	country_modifiers = {
+		workforce_scaled = {
+			country_military_tech_spread_mult = 0.01
+		}
+	}
+}
+
+pm_research_institute = {
+	texture = "gfx/interface/icons/production_method_icons/unused/engineering_university.dds"
+	building_modifiers = {
+		level_scaled = {
+			building_employment_academics_add = 100
+            building_employment_engineers_add = 100
+            building_employment_machinists_add = 150
+            building_employment_laborers_add = 100
+		}
+	}
+	building_modifiers = {
+		workforce_scaled = {
+			building_input_paper_add = 1
+			building_input_services_add = 0.5
+		}	
+	}
+	country_modifiers = {
+		workforce_scaled = {
+			country_production_tech_spread_mult = 0.01
+		}
+	}
+}
+
+pm_liberal_arts = {
+	texture = "gfx/interface/icons/production_method_icons/unused/social_science.dds"
+	building_modifiers = {
+		level_scaled = {
+			building_employment_academics_add = 150
+            building_employment_bureaucrats_add = 50
+            building_employment_clerks_add = 100
+            building_employment_laborers_add = 100
+		}
+	}
+	building_modifiers = {
+		workforce_scaled = {
+			building_input_paper_add = 0.5
+			building_input_services_add = 1
+		}	
+	}
+	country_modifiers = {
+		workforce_scaled = {
+			country_society_tech_spread_mult = 0.01
+		}
+	}
+}
 
+pm_general_studies = {
+	texture = "gfx/interface/icons/production_method_icons/unused/lightbulbs.dds"
+	building_modifiers = {
+		
+		level_scaled = {
+			building_employment_academics_add = 100
+            building_employment_engineers_add = 50
+            building_employment_officers_add = 50
+            building_employment_soldiers_add = 100
+            building_employment_clerks_add = 100
+            building_employment_laborers_add = 100
+		}
+	}
+	building_modifiers = {
+		workforce_scaled = {
+			building_input_paper_add = 0.5
+			building_input_small_arms_add = 0.5
+			building_input_services_add = 0.5
+		}	
+	}
+	country_modifiers = {
+		workforce_scaled = {
+			country_tech_spread_mult = 0.0033
+		}
+	}
+}
+pm_no_initiatives = {
+	texture = "gfx/interface/icons/production_method_icons/ownership_aristocrats.dds"
+}
+pm_urban_academia = {
+	texture = "gfx/interface/icons/production_method_icons/government_run.dds"
+	building_modifiers = {
+		
+		level_scaled = {
+			building_employment_academics_add = 25
+            building_employment_clerks_add = 75
+            building_employment_laborers_add = 100
+		}
+	}
+	building_modifiers = {
+		workforce_scaled = {
+			building_input_paper_add = 0.25
+			building_input_services_add = 0.25
+		}	
+	}
+	state_modifiers = {
+		workforce_scaled = {
+			state_education_access_add = 0.01
```

Comment:
this is way to high, that is 1% I meant 0.1% (so 0.001)

Reply:
My bad, I will fix it :)